SEP-0002.md

```diff
@@ -3,203 +3,288 @@
+Self-nominations are encouraged.
+
+The process for adding a new Project member is as follows:
+
+1. A nomination is received by the Steering Committee, a nomination should include a short, factual, statement stating how the nominee meets the criteria for Project Custodianship.
+The Steering Committee, and the Ombudsperson check that the nomination is factually accurate, their record of contribution to the community adheres to the Code of Conduct, and they wish to serve as a Project Custodian (if not self-nominated).
+1. The candidate is then put to the Project Custodians.
+The name and nomination statement of the candidate will be shared with the Project Custodians.
+The candidate can become a Project Custodian if, during the two weeks following the candidate being put to the Project Custodians, five existing Project Custodians approve the candidate and no Project Custodians call for an election.
+1. Project Custodians can call for an election either by informing the Steering Committee or the Ombudsperson.
+If a vote is called for then the candidate will become a Project Custodian with a simple majority of all votes cast in the election.
+An election needs to be announced to the Project Custodians one week before the voting opens and voting will be open for two weeks.
+1. The candidate will be informed promptly of the outcome by the Steering Committee.
+
+The name of a candidate should not be shared outside of the Project Custodians until they become a Project Custodian.
+A public list of Project Custodians will be kept on the SunPy website.
+
+Nominations, approvals and any elections of Project Custodians should take place before the initial election of the Steering Committee.
+
+#### Active and Emeritus Status
+
+Project Custodians have no term limits.
+Project Custodians who are no longer actively contributing to the project should contact the Steering Committee to move themselves to emeritus status.
+Active Project Custodians who have not participated in the project in the manner required for Project Custodianship for two years, will be contacted by the Steering Committee and be asked to move themselves to emeritus status.
+The Steering Committee can change a Project Custodian's status to emeritus.
+Emeritus Project Custodians will continue to be listed on the website.
+Emeritus Project Custodians are not able to participate in votes.
+An emeritus member may declare themselves to be active again at any time if they feel they are making contributions.
+
+#### Removing Project Custodians
+
+In exceptional circumstances, it may be necessary to remove someone from the Project Custodians against their will.
+A vote of the Project Custodians must be held to remove a Project Custodian.
+Such a vote is triggered by a motion (sent to the Steering Committee and the Ombudsperson) made by an active Project Custodian, which must be seconded by an additional Project Custodian.
```

Comment:
Per my comment above, it can be unclear whether "active" here is referring to status (i.e., non-emeritus) or merely to an engaged custodian.